From: Christian Melki <christian.melki@t2data.com>
Date: Mon, 4 Apr 2022 13:59:16 +0200
Subject: [PATCH] tls: Don't build TLS tests.

* Rework previous patch for glib-networking.

TLS tests fail to compile because gnutls is built without
pkcs11 support in ptxdist.

Signed-off-by: Christian Melki <christian.melki@t2data.com>
---
 tls/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tls/meson.build b/tls/meson.build
index e76e7666bb30..c96d145f6d8d 100644
--- a/tls/meson.build
+++ b/tls/meson.build
@@ -14,4 +14,3 @@ if backends.contains('openssl')
   subdir('openssl')
 endif
 
-subdir('tests')
